inst/doc/plot_geotiff.R

```diff
@@ -0,0 +1,46 @@
+## ----setup, include=FALSE---------------------------------
+library(rmarkdown)
+options(continue=" ")
+options(width=60)
+library(knitr)
+library(geoknife)
+
+
+## ---- warning=FALSE, message=FALSE------------------------
+library(geoknife)
+
+## ---------------------------------------------------------
+knife <- webprocess(algorithm = list('OPeNDAP Subset'="gov.usgs.cida.gdp.wps.algorithm.FeatureCoverageOPeNDAPIntersectionAlgorithm"))
+fabric <- webdata(url='dods://opendap.larc.nasa.gov/opendap/SortByProduct/CERES/EBAF/Surface_Edition2.8/CERES_EBAF-Surface_Edition2.8_200003-201502.nc', 
+                  variable="sfc_sw_down_all_mon", #Surface Shortwave Flux Down, Monthly Means, All-Sky conditions
+                  times=c('2014-07-15','2014-07-15'))
+stencil <- simplegeom(data.frame('point1' = c(-5,32), 'point2' = c(-90,-78))) # big 'ol chunk 'o data
+job <- geoknife(stencil, fabric, knife, wait = TRUE, OUTPUT_TYPE = "geotiff")
+
+## ---- message=FALSE---------------------------------------
+file <- download(job, destination = file.path(tempdir(), 'nasa_data.zip'), overwrite=TRUE)
+unzip(file, exdir=file.path(tempdir(),'NASA'))
+tiff.dir <- file.path(tempdir(),'NASA')
+
+## ---- warning=FALSE, message=FALSE------------------------
+library(rasterVis)
+nasa <- raster(file.path(tiff.dir , dir(tiff.dir)))
+
+## ---- warning=FALSE, message=FALSE------------------------
+library(ggmap)
+library(ggplot2)
+
+
+
+world <- map_data("world2")
```

Comment:
for both `rgdal` and `maps`, i'm suggesting the `library()` call not because it's necessary to load but because it will make sure the user has that package installed